Celeste.Mod.mm/Patches/Level.cs

```diff
@@ -582,8 +583,13 @@ private static void _GCCollect() {
         public override void End() {
             orig_End();
 
+            // reload the vanilla Portraits.xml in case it was overridden by a map
+            // else, if the Portraits.xml doesn't have portrait_madeline defined,
+            // the game will crash when trying to load a save file (since it shows madeline's portrait)
+            GFX.PortraitsSpriteBank = new SpriteBank(GFX.Portraits, Path.Combine("Graphics", "Portraits.xml"));
+
             // if we are not entering PICO-8 or the Reflection Fall cutscene...
-            if (!(patch_Engine.NextScene is Pico8.Emulator) && !(patch_Engine.NextScene is OverworldReflectionsFall)) {
+            if (patch_Engine.NextScene is not Pico8.Emulator && patch_Engine.NextScene is not OverworldReflectionsFall) {
```

Comment:
if you're changing this anyway, might as well make it `patch_Engine.NextScene is not (Pico8.Emulator or OverworldReflectionsFall)`

Reply:
###### already mentioned this in celestecord, but i'll put this here for consistency
I did try that at first, but that resulted in different IL. Fearing mod incompatibilities, I reduced the simplification to what I committed.

I could check for mod incompatibilities later and apply your suggestion if possible.

Reply:
![Wait, it's all On.* hooks? Always has been](https://github.com/user-attachments/assets/400a9f49-3faa-45aa-8758-d511475a45e9)

The suggestion has been applied.